### PR TITLE
Remove deprecated code #408 part 1

### PR DESCRIPTION
**Pull request by Neurrone**
Part 1 for fixing #408

### CODE REVIEW

**Comment by Neurrone** [comment]
@rahiel is there a way to run the tests on Windows? Having to rely on Travis to test it so I can find usage of deprecated methods.

**Comment by rahiel** [comment]
If you did the setup from the [contribution guide](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/.github/CONTRIBUTING.rst), you can run the tests with `make test` or `nosetests -v` in the project directory.

**Comment by Neurrone** [comment]
@rahiel thanks. I only saw the "make tests" command being mentioned; should I open a new issue/PR to include it in the contribution guide?

**Comment by rahiel** [comment]
I'll add a note to guide, you can focus on this PR :)

**Comment by Neurrone** [comment]
@rahiel ready for review :)

File: telegram/ext/dispatcher.py [hunk 1]
```diff
@@ -227,8 +226,8 @@ def stop(self):
                 sleep(0.1)
             self.__stop_event.clear()
 
-        # async threads must be join()ed only after the dispatcher thread was joined,
-        # otherwise we can still have new async threads dispatched
+# async threads must be join()ed only after the dispatcher thread was joined,
+# otherwise we can still have new async threads dispatched
```
Comment (rahiel) [inline telegram/ext/dispatcher.py]: Why did you misalign the comments here? Same at line 280

Comment (Neurrone) [inline telegram/ext/dispatcher.py]: Oh, that must have been the pre-commit hooks. I don't remember making the change.

Comment (rahiel) [inline telegram/ext/dispatcher.py]: That's weird, but our pre-commit hooks have given us lots of trouble already...
Could you fix the alignment and skip the pre-commit hooks by doing `git commit --no-verify`?
Everything else looks good, will be ready to merge after this.

**Comment by Neurrone** [comment]
@rahiel fixed.

**Comment by rahiel** [comment]
Thanks for you contribution @Neurrone! Your PR is now ready to be merged.

**Comment by jh0ker** [comment]
Excellent :) Thank you for your contribution!

**Pull request merged**